packages/common/src/ui/layout/tables/columns/CheckboxSelectionColumn.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Checkbox } from '../../../components/inputs/Checkbox';
+import { Column } from 'react-table';
+
+export const getCheckboxSelectionColumn = (): Column => ({
+  id: 'selection',
+  Header: ({ getToggleAllRowsSelectedProps }) => (
```

Comment:
Looks like maybe there isn't a header with a checkbox like this and it'd actually be in some other component in the toolbar above, but left it for now